src/command_modules/azure-cli-billing/azure/cli/command_modules/billing/__init__.py

```diff
@@ -3,12 +3,27 @@
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
 
+from azure.cli.core import AzCommandsLoader
+
 import azure.cli.command_modules.billing._help  # pylint: disable=unused-import
 
 
-def load_params(_):
-    import azure.cli.command_modules.billing._params  # pylint: disable=redefined-outer-name, unused-variable
+class BillingCommandsLoader(AzCommandsLoader):
+
+    def __init__(self, cli_ctx=None):
+        from azure.cli.core.sdk.util import CliCommandType
+        billing_custom = CliCommandType(operations_tmpl='azure.cli.command_modules.billing.custom#{}')
+        super(BillingCommandsLoader, self).__init__(cli_ctx=cli_ctx, custom_command_type=billing_custom,
+                                                    min_profile="latest")
```

Comment:
Don't use min_profile='latest'.  Otherwise it will magically disappear from a profile that supports it once latest moves on.  Instead, use the previous profile, but add one to the day.  Make sure you test on both profiles to ensure it appears/doesn't appear as required.